js/main.js

```diff
@@ -1,8 +1,17 @@
 const savebtn = document.getElementById('savebtn');
```

Comment:
Do we need to think about writing case style? 

https://en.wikipedia.org/wiki/Naming_convention_(programming)#:~:text=In%20computer%20programming%2C%20a%20naming,in%20source%20code%20and%20documentation.